### ISSUE TITLE
Out of range: End of sequence

### ISSUE THREAD

**Issue by gameking157** [tensorflow/tensorflow#15769]
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.4.1
- **Python version**:   python3.5
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:   Cuda 8.0  cudnn6_6.0.
- **GPU model and memory**:  Titax 12G
- **Exact command to reproduce**:

python -m nmt.nmt \
    --src=vi --tgt=en \
    --vocab_prefix=/tmp/nmt_data/vocab  \
    --train_prefix=/tmp/nmt_data/train \
    --dev_prefix=/tmp/nmt_data/tst2012  \
    --test_prefix=/tmp/nmt_data/tst2013 \
    --out_dir=/tmp/nmt_model \
    --num_train_steps=12000 \
    --steps_per_stats=100 \
    --num_layers=2 \
    --num_units=128 \
    --dropout=0.2 \
    --metrics=bleu

2018-01-01 21:14:49.071267: W tensorflow/core/kernels/lookup_util.cc:362] Table trying to initialize from file /tmp/nmt_data/vocab.vi is already initialized.
2018-01-01 21:14:49.071378: W tensorflow/core/kernels/lookup_util.cc:362] Table trying to initialize from file /tmp/nmt_data/vocab.en is already initialized.
2018-01-01 21:14:49.071554: W tensorflow/core/kernels/lookup_util.cc:362] Table trying to initialize from file /tmp/nmt_data/vocab.en is already initialized.
  loaded infer model parameters from /tmp/nmt_model/translate.ckpt-9000, time 0.08s

  decoding to output /tmp/nmt_model/output_dev.
2018-01-01 21:14:58.585992: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: End of sequence
         [[Node: IteratorGetNext = IteratorGetNext[output_shapes=[[?,?], [?]], output_types=[DT_INT32, DT_INT32], _device="/job:localhost/replica:0/task:0/device:CPU:0"](Iterator)]]
2018-01-01 21:14:58.586255: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: End of sequence
         [[Node: IteratorGetNext = IteratorGetNext[output_shapes=[[?,?], [?]], output_types=[DT_INT32, DT_INT32], _device="/job:localhost/replica:0/task:0/device:CPU:0"](Iterator)]]
  done, num sentences 1553, num translations per input 1, time 9s, Mon Jan  1 21:14:58 2018.
  bleu dev: 5.2
  saving hparams to /tmp/nmt_model/hparams

  decoding to output /tmp/nmt_model/output_test.
2018-01-01 21:15:08.289098: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: End of sequence
         [[Node: IteratorGetNext = IteratorGetNext[output_shapes=[[?,?], [?]], output_types=[DT_INT32, DT_INT32], _device="/job:localhost/replica:0/task:0/device:CPU:0"](Iterator)]]
2018-01-01 21:15:08.289145: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: End of sequence
         [[Node: IteratorGetNext = IteratorGetNext[output_shapes=[[?,?], [?]], output_types=[DT_INT32, DT_INT32], _device="/job:localhost/replica:0/task:0/device:CPU:0"](Iterator)]]
  done, num sentences 1268, num translations per input 1, time 9s, Mon Jan  1 21:15:08 2018.
  bleu test: 4.4

**Comment by mohantym** [comment]
Hi @gameking157 ! Sorry for the late response.
It seems you are using older versions(1.x versions) of Tensorflow which is not supported any more. You can migrate to 2.x version using this [document](https://www.tensorflow.org/guide/migrate/migrate_tf2) ? Thanks!
